setup.py

```diff
@@ -44,6 +45,22 @@ def _export_version(version, sha):
         fileobj.write("git_version = {}\n".format(repr(sha)))
 
 
+def check_submodules():
+    third_party_dir = ROOT_DIR / 'third_party'
+    if not (third_party_dir / 're2' / 'CMakeFiles.txt').exists():
+        print(' --- third_party/re2/CMakeFiles.txt do not exist, trying to init submodules')
+        try:
```

Comment:
[PyTorch fails and asks the user to do it manually](https://github.com/pytorch/pytorch/blob/master/setup.py#L296-L299). I worry that this might cause issues when someone wants to debug / modify the submodules and then subsequently runs setup.py. Are there similar plans for PyTorch as well?

Reply:
@cpuhrsch yes, I plan to make similar change for pytorch as well, although Pytorch is less likely to be built directly from source (building TorchText using direct link takes less than 15 min, whereas for PyTorch, especially with GPU support will likely to take more than an hour)

Reply:
What is the context of this work? Are we going to do the same for torchaudio (which I support)?